en/config/safety.md

```diff
@@ -20,18 +20,20 @@ Action | Description
 <a id="action_hold"></a>[Hold mode](../flight_modes/hold.md) | The vehicle will enter *Hold mode*. For multicopters this means the vehicle will hover, while for fixed/wing the vehicle will circle.
 <a id="action_return"></a>[Return mode](../flight_modes/return.md) | The vehicle will enter *Return mode*. Return behaviour can be set in the [Return Home Settings](#return-mode-settings) (below).
 <a id="action_land"></a>[Land mode](../flight_modes/land.md) | The vehicle will enter *Land mode*, and lands immediately.
+<a id="action_disarm"></a>Disarm | Stops the motors immediately.
```

Comment:
How does this differ from "Lockdown" option below?

Reply:
More precisely, I see these options in the simulation:
![image](https://user-images.githubusercontent.com/5368500/193949368-ce318962-5976-4eb2-826f-eb2e3b7149d3.png) 
![image](https://user-images.githubusercontent.com/5368500/193949490-e7fbfa43-53aa-4615-83b5-5fa1f7843bbc.png)

So would I be correct in saying:
- There is no "lockdown action".
- Terminate is flight termination, disarm is disarm. They both disarm the motors, but flight termination also sets failsafe values. FMI, why would you need both?

Reply:
PS IFF this is the same result/danger as termination is it protected by the circuit breaker? (CBRK_FLIGHTTERM)

Reply:
I'm removing the lockdown action and replace it with disarming, as this should be easier to understand.
Lockdown is currently still used for the kill switch, so it's not removed completely.

Reply:
@bkueng So to be clear:
1. Lockdown is not a failsafe action - it exists for the kill switch but is not something you can select as an action for any failsafe. I will therefore remove it.
2. The term "disarm" is certainly more clear than "lockdown"
3. Disarm turns off motors, while flight termination sets stuff to it's failsafe values (i.e. turns off motors, but might deploy stuff)
   - Is that right?
   - IF so, why is this considered less severe than flight termination? Is it because disarming is revertable? 
   - This is just as dangerous as flight termination right - the vehicle drops out of the air. Is it interlocked by CBRK_FLIGHTTERM or some other circuit breaker?
   - Can flight termination do other stuff, like quadchute? My assumption is NO.

Reply:
Correct.

> Is it interlocked by CBRK_FLIGHTTERM or some other circuit breaker?

No, you need to actively select it in the first place.

> Can flight termination do other stuff, like quadchute? My assumption is NO.

No